include/powsybl/iidm/converter/xml/Constants.hpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_CONSTANTS_HPP
+#define POWSYBL_IIDM_CONVERTER_CONSTANTS_HPP
```

Comment:
Guardian is not OK: It's already use by the Constants.hpp in converter. Maybe you should move this constant somewhere else (ImportOptions and ExporterOptions)?

Reply:
You're right, the guard should be 
```
#ifndef POWSYBL_IIDM_CONVERTER_XML_CONSTANTS_HPP
```

as this file is in include\powsybl\iidm\converter\xml